MAGEDOC-3969 Added note to MTF to use MFTF (#5054)
magento/devdocs: guides/v2.2/mtf/mtf_installation.md

title: Installation of the Functional Testing Framework Entities
---

{: .bs-callout-info}

> The Magento Testing Framework (MTF) has been superseded by the Magento Functional Testing Framework ([MFTF][]).

Reviewer: ```suggestion
The Magento Testing Framework (MTF) is superseded by the Magento Functional Testing Framework ([MFTF][]).
```

> While the MTF is still functional, all MTF tests are being ported over to the MFTF. We recommend using the MFTF for testing.

Reviewer: ```suggestion
While the MTF is still functional, all MTF tests are ported over to the MFTF. We recommend using the MFTF for testing.
```

Reviewer: Keep those "be" verbs outta here!

Reviewer: Please make these changes on the other version pages in this PR.

Author: Will do.

Author: Actually, the are still in the process of porting. It's a work in progress.
